src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -245,6 +230,13 @@ ov::npuw::LLMCompiledModel::LLMCompiledModel(const std::shared_ptr<ov::Model>& m
     std::map<std::string, ov::Any> npuw_llm_props;
     std::map<std::string, ov::Any> other_props;
     split_llm_properties(properties, npuw_llm_props, other_props);
+
+    // Remove "NPUW_LLM_PREFILL_CONFIG", "NPUW_LLM_GENERATE_CONFIG" from map,
+    // to not pass them into ::intel_npu::Config object, as we don't need to
+    // preserve them somewhere.
+    auto prefill_config_opt = pop_option(npuw_llm_props, std::string("NPUW_LLM_PREFILL_CONFIG"));
+    auto generate_config_opt = pop_option(npuw_llm_props, std::string("NPUW_LLM_GENERATE_CONFIG"));
+
     m_cfg.update(any_copy(npuw_llm_props));
```

Comment:
I believe  nothing from `npuw_llm_props` should get into `m_cfg`, right?

Everything related to LLM pipeline can be extracted here and then forgotten.

Reply:
That is true, however how would you check and test what have you passed?

Reply:
The same way as it's done in GenAI. You can put any checks within this file...

Reply:
Got it! Discussed internally